lib/models/device.coffee

```diff
@@ -1361,6 +1361,42 @@ getDeviceModel = (deps, opts) ->
 						force: Boolean(options.force)
 		.asCallback(callback)
 
+	###*
+	# @summary Get the target supervisor state on a device
+	# @name getSupervisorTargetState
+	# @public
+	# @function
+	# @memberof balena.models.device
+	#
+	# @param {String|Number} uuidOrId - device uuid (string) or id (number)
+	# @returns {Promise}
+	#
+	# @example
+	# balena.models.device.getSupervisorTargetState('7cf02a6').then(function(state) {
+	#	 	console.log(state);
+	# });
+	#
+	# @example
+	# balena.models.device.getSupervisorTargetState(123).then(function(state) {
+	#	 	console.log(state);
+	# });
+	#
+	# @example
+	# balena.models.device.getSupervisorTargetState('7cf02a6', function(error, state) {
+	# 		if (error) throw error;
+	#	 	console.log(state);
```

Comment:
Let's change the examples to not use 2 tabs for their indentation.

Reply:
Done! That was consistent with the `getSupervisorState` method, but I fixed that too.